topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -46,8 +50,8 @@ async def load_metadata(self, asset: Asset) -> None:
                 "linz:photo_no": asset_metadata["photo_no"],
                 "linz:alternate_survey_name": asset_metadata["alternate_survey_name"],
                 "linz:camera": asset_metadata["camera"],
-                "linz:camera_sequence_no": asset_metadata["camera_sequence_no"],
-                "linz:nominal_focal_length": asset_metadata["nominal_focal_length"],
+                "camera:sequence_number": asset_metadata["camera_sequence_no"],
```

Comment:
also possibly super pedantic, but it might be nice to do a separate `update` per extension, maybe a `add_camera_metadata()` or something